engine/src/main/java/org/hibernate/search/engine/cfg/spi/ConfigurationPropertySourceScopeUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.cfg.spi;
```

Comment:
It was here so `HibernateSearchCompositeMappingProducer` could access it. now that you've changed the `HibernateSearchCompositeMappingProducer` and it doesn't need the config providers -- this can go back to the impl package